x-pack/plugins/ml/common/types/data_frame_analytics.ts

```diff
@@ -59,13 +59,19 @@ export type AnalysisConfig = estypes.MlDataframeAnalysisContainer;
 export interface DataFrameAnalyticsConfig
   extends Omit<estypes.MlDataframeAnalyticsSummary, 'analyzed_fields'> {
   analyzed_fields?: estypes.MlDataframeAnalysisAnalyzedFields;
+  _meta?: Record<string, any>;
```

Comment:
```suggestion
  _meta?: Record<string, unknown>;
```

Reply:
Updated in 642ce6dbe70bb719b6416e5a61184a6b966b71d7